odl/test/operator/pspace_ops_test.py

```diff
@@ -54,6 +54,55 @@ def test_pspace_op_init(base_op):
     assert op.range == A.range ** 2
 
 
+def test_pspace_op_derivative(base_op):
+    """Test derivatives with different base operators.
+
+    See also https://github.com/odlgroup/odl/issues/1290
```

Comment:
Yeah, I'll remove this.

Could you please either approve or request changes? If you just comment, it will still say "Review required" even though the review just happened.